docker/cleanup_ns3.sh

```diff
@@ -3,11 +3,11 @@ set -e
 
 mkdir -p sims/external/ns-3-new/
 cd sims/external/ns-3
-find build -name '*.o' -delete
 cp -r \
   build/ \
-  cosim-run.sh \
-  cosim-dctcp-run.sh \
+  simbricks-run.sh \
+  ns3 \
+  .lock-ns3* \
   ../ns-3-new/
```

Comment:
What are the .lock-ns3* files here about?

Reply:
You usually should have only one. On a Linux machine you will have `.lock-ns3_linux_build`. The file is basically a python script that contains variable definitions holding information about the build. This includes configurations like the build profile, but also other useful information like a list of all runnable programs that were compiled during the build. This is used by the `ns3` helper python script to offer part of its functionality, which is why we have to keep it.

Reply:
Makes sense, thanks.